modules/creating-a-machine-pool-cli-win-li.adoc

```diff
@@ -0,0 +1,101 @@
+// Module included in the following assemblies:
+//
+// * rosa_cluster_admin/rosa_nodes/rosa-managing-worker-nodes.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="creating_a_machine_pools_cli_win_li_{context}"]
+= Creating a machine pool with AWS Windows License Included-enabled using the {rosa-cli}
+
+[role="_abstract"]
+If you are using {VirtProductName} on a {product-title} cluster, you might want to run Windows VMs. In order to be license-compliant with Microsoft Windows in AWS, the hosts (metal EC2 instances) running these VMs must be enabled with AWS EC2 Windows License Included (LI).
+
+[IMPORTANT]
+====
+Enabling Windows LI on a machine pool immediately applies the associated licensing fees on that specific machine pool. This includes billing for the full vCPU allocation of each Windows-LI-enabled host in your {product-title} cluster. For more information, see link:https://aws.amazon.com/windows/resources/licensing//[Microsoft Licensing on AWS].
+====
+
+.Prerequisites
+
+* You installed and configured the {rosa-cli} version 1.2.58 or above.
+* You logged in to your Red{nbsp}Hat account using the {rosa-cli}.
+* You created a {product-title} cluster version 4.19 or above.
+* You identified a bare-metal EC2 instance type to use {VirtProductName}. For more information about bare-metal instance types, see link:https://aws.amazon.com/ec2/instance-types/#Bare_metal[Amazon EC2 Instance Types].
+* You are in compliance with Microsoft and AWS requirements for the Microsoft licenses and associated costs.
+
+
+.Procedure
+
+* To add a Windows LI-enabled machine pool to a {product-title} cluster, create the machine pool with the following definitions.
++
+[source,terminal]
+----
+$ rosa create machinepool --cluster=<cluster-name> \
+                          --name=<machine_pool_id> \
+                          --replicas=<replica_count> \
+                          --instance-type=<instance_type> \
+                          --type=<image_type>
+----
+where:
+
+`--name=<machine_pool_id>`:: Specifies the name of the machine pool. Replace `<machine_pool_id>` with the name of your machine pool.
+`--replicas=<replica_count>`:: Specifies the number of compute nodes to provision. If you deployed {product-title} using a single availability zone, this defines the number of compute nodes to provision to the machine pool for the zone. If you deployed your cluster using multiple availability zones, this defines the number of compute nodes to provision in total across all zones and the count must be a multiple of 3. The `--replicas` argument is required when autoscaling is not configured.
+`--instance-type=<instance_type>`:: Specifies the instance type. You can only select a bare-metal instance type to enable Windows LI. For example, you can use `m5zn.metal` or `i3.metal`.
+`--type=<type>`:: You must specify `Windows` to ensure the machine pool is created with Windows LI enabled.
```

Comment:
here we do not use Windows LI-enabled. No `-` here. Just to be consistent remove or leave `-`, thanks